torch/csrc/utils/disable_torch_function.cpp

```diff
@@ -125,3 +126,111 @@ PyObject* THPModule_disable_torch_function(PyObject *self, PyObject *a) {
   return result;
   END_HANDLE_TH_ERRORS
 }
+
+// Makes sure that we don't check for __torch_function__ on basic Python types
+static bool is_basic_python_type(PyTypeObject *tp)
```

Comment:
This function looks unchanged but the functions below seem to have been changed

Reply:
Indeed, this one was just moved from `torch/csrc/utils/python_arg_parser.h` The functions below were modified to:
a) Bail out faster. (e.g. don't do attr checks on known types)
b) More efficiently handle checking of multiple Python values, which generally means trying to be as lazy as possible with Python containers. (e.g. `PySequence_Fast` does an extra refcount bump and decref because it has no way of knowing that we'll keep args alive until it's done.)